.github/workflows/codeql.yml

```diff
@@ -0,0 +1,90 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL Advanced for release"
+
+on:
+  push:
+    branches: [ "release/2024.05" ]
```

Comment:
Suppose we will check the "master" branch, right? PR#11 will be merged to master branch after review/approval.